chef/cookbooks/hyperv/attributes/default.rb

```diff
@@ -2,11 +2,31 @@
 # Windows features required:
 #
 
-default[:features_list][:hyperv] = ["Microsoft-Hyper-V", "Microsoft-Hyper-V-Management-PowerShell"]
-default[:features_list][:management] = ["Remote-Desktop-Services"]
-default[:features_list][:iscsi_target] = ["File-Services", "CoreFileServer"]
+default[:features_list][:hyperv] = {
+  "Microsoft-Hyper-V" => {
+    "restart" => false
+  },
+  "Microsoft-Hyper-V-Management-PowerShell" => {
+    "restart" => true
```

Comment:
The old code was triggering a reboot at the end, after installing all features, but the idea is that we need to be able to reboot per feature if that is required. Else the code will try to use what was installed by the feature and that is not available until after the reboot.